app/src/main/res/layout/contactpatient_callpatient.xml

```diff
@@ -10,6 +10,68 @@
   tools:context="org.simple.clinic.contactpatient.views.CallPatientView"
   tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout">
 
+  <com.google.android.material.card.MaterialCardView
+    android:id="@+id/callResultOutcomeCardView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/spacing_16"
+    android:elevation="@null"
+    android:visibility="gone"
+    app:cardBackgroundColor="@color/simple_green_100"
+    app:cardCornerRadius="@dimen/spacing_4"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    app:strokeColor="@color/simple_green_600"
+    app:strokeWidth="@dimen/spacing_1">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginHorizontal="@dimen/spacing_16"
+      android:layout_marginVertical="@dimen/spacing_12">
+
+      <ImageView
+        android:id="@+id/callResultOutcomeIcon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_check_circle_24px"
```

Comment:
Same comment as above. Since we are setting this content at runtime, let's change this to be a `tools` attr instead.